charity_finder/views.py

```diff
@@ -11,32 +12,58 @@
 # Create your views here.
 def home(request):
 
-    m = folium.Map(location=[59.09827437369457, 13.115860356662202], zoom_start=3)
+    project_map = get_map()
 
-    # TODO: have smaller goals but then limit it to continent or country
-    projects = Project.objects.filter(goal_remaining__gte=500_000).values(
-        "title", "project_link", "latitude", "longitude", "goal_remaining"
+    context = {
+        "project_map": project_map,
+    }
+    return render(request, "home.html", context)
+
+
+def get_map():
+
+    # Filtering by remaining funding money by Region
+    # TODO:  Include more regions
+    goal_limit = 100000
```

Comment:
You can use `_` to make big numbers more readable, e.g. `100_000`, also I usually put constants like this at module level at the top of this file

Reply:
`GOAL_LIMIT: final = 100_000` -> you can even add a type hint in there stating that we should not override it. You still can but mypy would catch it